pipeline/clean/requirements/clean.in

```diff
@@ -1,4 +1,5 @@
-opuscleaner==0.4.0
+# Add support for bs, id, sr, tr, vi.
+git+https://github.com/gregtatum/OpusCleaner.git@spring2024
```

Comment:
Even as a temporary solution, I would prefer to merge it to our Mozilla fork https://github.com/mozilla/OpusCleaner and use it from there.

Reply:
How do I get access to this repo? Can I directly push to it?

Reply:
I added our team there and also at https://github.com/mozilla/OpusTrainer. So feel free to sync it and PR there.

Reply:
Or directly push, up to you

Reply:
I directly pushed to spring2024.